Marlin/src/lcd/dogm/status_screen_DOGM.cpp

```diff
@@ -72,7 +72,7 @@ FORCE_INLINE void _draw_centered_temp(const int16_t temp, const uint8_t tx, cons
 
 #define DO_DRAW_BED (HAS_HEATED_BED && STATUS_BED_WIDTH && HOTENDS <= 3 && DISABLED(STATUS_COMBINE_HEATERS))
 #define DO_DRAW_FAN (HAS_FAN0 && STATUS_FAN_WIDTH && STATUS_FAN_FRAMES)
-//#define DO_DRAW_CHAMBER (HAS_TEMP_CHAMBER && ((HOTENDS <= 2 && DO_DRAW_BED) || (!DO_DRAW_BED && HOTENDS <= 3)))
+#define DO_DRAW_CHAMBER (HAS_TEMP_CHAMBER && ((HOTENDS <= 2 && DO_DRAW_BED) || (!DO_DRAW_BED && HOTENDS <= 3)))
```

Comment:
This was disabled because currently chamber icon is displayed over the bed icon...
I think that X may be left of bed (if possible) or over bed location when this is not present

Reply:
Fixing all this